crates/client-api/src/routes/database.rs

```diff
@@ -907,10 +886,8 @@ pub struct SetNameQueryParams {
 pub async fn set_name<S: ControlStateDelegate>(
     State(ctx): State<S>,
     Query(SetNameQueryParams { domain, address }): Query<SetNameQueryParams>,
-    auth: SpacetimeAuthHeader,
+    Extension(auth): Extension<SpacetimeAuth>,
 ) -> axum::response::Result<impl IntoResponse> {
-    let auth = auth_or_unauth(auth)?;
```

Comment:
(review note: I think this used to fail if auth wasn't present, but it's fine because we check for a specific identity below)